addons/mail/security/mail_security.xml

```diff
@@ -276,4 +276,32 @@
             <field name="perm_create" eval="True"/>
             <field name="perm_unlink" eval="True"/>
         </record>
+
+        <record id="ir_rule_mail_canned_response_admin_rule" model="ir.rule">
+            <field name="name">Canned response: admin access</field>
+            <field name="model_id" ref="model_mail_canned_response"/>
+            <field name="groups" eval="[Command.link(ref('group_mail_canned_response_admin'))]"/>
+            <field name="perm_read" eval="False"/>
+        </record>
+
+        <record id="ir_rule_mail_canned_response_creator_rule_cwu" model="ir.rule">
+            <field name="name">Canned response: User access</field>
+            <field name="model_id" ref="model_mail_canned_response"/>
+            <field name="groups" eval="[Command.link(ref('base.group_user'))]"/>
+            <field name="domain_force">[('create_uid', '=', user.id)]</field>
+            <field name="perm_write" eval="False"/>
+            <field name="perm_create" eval="False"/>
+            <field name="perm_unlink" eval="False"/>
```

Comment:
There is currently no rule to restrict users from writing/creating/unlinking any canned response, is that intended?